modules/consensus/synchronize_test.go

```diff
@@ -108,7 +107,6 @@ func TestBlockHistory(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow()
 	}
-	t.Parallel()
```

Comment:
Oh I thought it was `TestBlockHistory`. Weird, it must be a bug in GitHub.

![screen shot 2016-06-20 at 9 20 22 pm](https://cloud.githubusercontent.com/assets/5420909/16215092/d4346366-372c-11e6-957a-9a9361ea47ce.png)